toolkit/committee-selection/query/src/commands.rs

```diff
@@ -1,6 +1,11 @@
+//! Implementations for CLI query commands for getting Ariadne params and registration status
 use crate::SessionValidatorManagementQueryApi;
 use sidechain_domain::{McEpochNumber, StakePoolPublicKey};
 
+/// Returns Ariadne parameters in JSON format
+/// Arguments:
+///  - query: query api implementing [SessionValidatorManagementQueryApi] trait
+///  - mc_epoch_number: mainchain epoch for which to get Ariadne parameters
```

Comment:
If I query for epoch 800, will it return Ariadne Parameters that are set in epoch 800 or that are used by committee selection during Cardano epoch 800?